script/installation/packages.sh

```diff
@@ -104,14 +104,14 @@ if [ "$DISTRO" = "UBUNTU" ]; then
             echo -e "\n# Added by Peloton 'packages.sh' script on $(date)\ndeb $LLVM_PKG_URL $LLVM_PKG_TARGET" | sudo tee -a /etc/apt/sources.list > /dev/null
         fi
         sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 15CF4D18AF4F7421
-        sudo apt-get update -qq
         CMAKE_NAME="cmake3"
         FORCE_Y="--force-yes"
     else
         CMAKE_NAME="cmake"
         FORCE_Y=""
     fi
 
+    sudo apt-get update -qq
```

Comment:
The manpage of `apt` says

> you should never use -qq without a no-action modifier such as -d, --print-uris or -s as APT may decide to do something you did not expect. Configuration Item: quiet.

This was actually a problem in #1155, but I think we didn't fix/change this. If we have a better idea for this, we should change it everywhere.

Reply:
We can try `-qy` instead of `-qq`. 
`-q` single q: a little bit more output
`-y` still answering all questions with yes

Reply:
I don't think ```apt-get update``` will ever prompt for input, will it?

Reply:
@crd477 I think you are right.

Reply:
A single ```q``` should be fine then.  I only added the second one to squeeze even more useless information out of the output.
Which reminds me - in my opinion, it would be good to change the ```wget``` invocations added as part of the TF integration to ```wget -q``` for the same reason.